py-polars/polars/expr/expr.py

```diff
@@ -2955,18 +2955,25 @@ def n_unique(self) -> Self:
         """
         Count unique values.
 
+        Notes
+        -----
+        `null` is considered to be a unique value for the purposes of this count.
```

Comment:
"This count" sounds weird, maybe this is better?

```suggestion
        `null` is considered to be a unique value for the purposes of this operation.
```

Reply:
Works for me ;)